README.md

```diff
@@ -1 +1,60 @@
 # Core Domain Charts
+
+Core Domain Charts help you to visualise the strategic importance of each sub-domain or business capability in your software architecture allowing you to make business model-aligned architectural decisions.
+
+[Core Domains]() are the parts of your domain where the expected ROI curve is greatest, and deserve the highest focus.
+
+The true power of this technique is the conversations that it triggers, especially cross-discipline. Complexity is something that engineers can gauge whereas business differentiation is provided by product managers or business stakeholders.
+
+![alt text](resources/core-domain-chart-template.jpg "Core Domain Chart")
+
+## How to Use
+There are a variety of ways the Core Domain Chart can be used, but it's important not to try to and visualise all of the possible information in a single diagram. Below are three flavours to choose from.
```

Comment:
I am not sure the word flavour fits here, might be a bit confusing. There are more levels of detail added?

Reply:
I simplified. Thanks